releasenotes/notes/56845-inference-extension.yaml

```diff
@@ -7,4 +7,4 @@ issue:
 releaseNotes:
 - |
   **Added** support for Gateway API Inference Extension https://gateway-api-inference-extension.sigs.k8s.io/.
-  This feature is off by default and can be turned on with the `SUPPORT_GATEWAY_API_INFERENCE_EXTENSION` environment variable.
+  This feature is off by default and can be turned on with the `ENABLE_GATEWAY_API_INFERENCE_EXTENSION` environment variable.
```

Comment:
Which value is supposed to go in this variable?

Maybe `... and can be turned on by setting the environment variable ENABLE_GATEWAY_API_INFERENCE_EXTENSION to XYZ`?